app.py

```diff
@@ -1,16 +1,16 @@
 import logging
 import os
 
-from app_utils.sections.chat import show_chat_is_running_dialog
+from llm_studio.app_utils.sections.chat import show_chat_is_running_dialog
```

Comment:
Like the idea to move `app_utils` to llm_studio! Is this required for the tests?
Also, in a recent discussion with @fatihozturkh2o, we where thinking of removing the utils part from `app_utils` folder or renaming it to something like `frontend`. WDYT?

Reply:
it's required for getting the test coverage right. I would also add this to the command. Currently looking into best ways for the report.

Reply:
yeah, this is what we planned pretty much:

https://github.com/h2oai/h2o-llmstudio/pull/364#discussion_r1295726033

Reply:
we also need to move the train.py and train_wave.py to the code subfolder for discoverability in pytest-coverage.
I'll push this and add symlinks probably to keep the old behavior

Reply:
moving `train.py` and the other python files in the root dir is a bit more tricky than expected. I'd probably postpone for later, as it really only affects the coverage report.